pkgs/by-name/va/vanguards/package.nix

```
@@ -0,0 +1,48 @@
+{ pkgs, lib }:
```

```suggestion
{ lib }:
```
Please put all dependencies explicitly.